api/src/env.ts

```diff
@@ -161,6 +161,13 @@ const allowedEnvironmentVars = [
 	'MESSENGER_REDIS_HOST',
 	'MESSENGER_REDIS_PORT',
 	'MESSENGER_REDIS_PASSWORD',
+	// synchronization
+	'SYNCHRONIZATION_STORE',
+	'SYNCHRONIZATION_NAMESPACE',
+	'SYNCHRONIZATION_REDIS',
+	'SYNCHRONIZATION_REDIS_HOST',
+	'SYNCHRONIZATION_REDIS_PORT',
+	'SYNCHRONIZATION_REDIS_PASSWORD',
```

Comment:
**excessive | ikˈsesiv |**
_adjective_

more than is necessary, normal, or desirable; immoderate: he was drinking excessive amounts of brandy.

---

In all seriousness though: yes. Lets finalize this REDIS setup cause this is getting a little redicilous 😂